Sources/StreamChatUI/Common Views/ChatAvatarView_Tests.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatAvatarView_Tests: XCTestCase {
+    var avatarView: ChatAvatarView! = ChatAvatarView().withoutAutoresizingMaskConstraints
+
+    func test_default_avatarView_withImage_looksAsExpected() {
+        let testImage = TestImages.vader.image
+        avatarView.imageView.image = testImage
+        AssertSnapshot(avatarView)
+    }
+
+    func test_default_avatarView_withoutImage_looksAsExpected() {
+        // What is default behavior of AvatarView when no image?
+        // Shouldn't be there some default placeholder image?
+        AssertSnapshot(avatarView)
+    }
+}
```

Comment:
This file should be deleted, `ChatAvatarView` is superclass which is not used directly in our codebase.